indexer/osm_editor.hpp

```diff
@@ -209,7 +209,8 @@ class Editor final : public MwmSet::Observer
   void Invalidate();
 
   // Saves a feature in internal storage with FeatureStatus::Obsolete status.
-  void MarkFeatureAsObsolete(FeatureID const & fid);
+  bool MarkFeatureAsObsolete(FeatureID const & fid);
```

Comment:
Please check other places where this method is called to find out if you need to handle false return value.